labonneboite/importer/jobs/extract_etablissements.py

```diff
@@ -85,6 +88,7 @@ def get_sirets_from_database(self):
 
     @timeit
     def update_updatable_offices(self):
+        # FIXME parallelize and/or batch for better performance
```

Comment:
I'm not sure parallelization would make things any faster... This piece of code is limited by the db performance.